recipe/meta.yaml

```
@@ -99,7 +99,7 @@ requirements:
     # Resolves https://github.com/man-group/ArcticDB/issues/465
     - aws-crt-cpp >=0.19.8
     - prometheus-cpp
-    - protobuf
+    - protobuf >=3.5.0.post1, <5
```

I am not sure conda-build supports specification with `post` segment. If it does not, let's use:
```suggestion
    - protobuf >=3.5.1, <5
```